### ISSUE TITLE
[android] Build failing

### ISSUE THREAD

**Issue by AbijahKaj** [AzzappApp/react-native-skia-video#11]
Hello!
I'm trying to build for android but the build is failing with this error.

```
> Task :azzapp_react-native-skia-video:configureCMakeDebug[arm64-v8a] FAILED
C/C++: CMake Error at CMakeLists.txt:17 (add_library):
C/C++:   Target "react-native-skia-video" links to target
C/C++:   "react-native-reanimated::worklets" but the target was not found.  Perhaps
C/C++:   a find_package() call is missing for an IMPORTED target, or an ALIAS target
C/C++:   is missing?
```

My setup
- React native - 0.74.5
- Expo - 51
- Building on macOS

**Comment by AbijahKaj** [comment]
Updating reanimated to ` "react-native-reanimated": "3.15.0",` seems to fix the problem , but I still can't build.

I have a new error
```
Task :azzapp_react-native-skia-video:buildCMakeDebug[arm64-v8a] FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':azzapp_react-native-skia-video:buildCMakeDebug[arm64-v8a]'.

....

  [6/13] Building CXX object CMakeFiles/react-native-skia-video.dir/cpp/VideoFrame.cpp.o
  FAILED: CMakeFiles/react-native-skia-video.dir/cpp/VideoFrame.cpp.o 

            auto string = static_ref_cast<jni::JString>(error);
                 ^
  1 warning generated.
  ninja: build stopped: subcommand failed.

```

**Comment by eyu0415** [comment]
That's not error just warning.
I guess the android minSdkVersion required >= 26
The example in this repo has minSdkVersion target 28
you can set version in expo app.json with [expo-build-properties](https://docs.expo.dev/versions/latest/sdk/build-properties/#pluginconfigtypeandroid)

or
```gradle
    // project/android/build.gradle
    minSdkVersion = Integer.parseInt(findProperty('android.minSdkVersion') ?: '23')  // change 26
```

**Comment by fdecampredon** [comment]
Should be fixed in 0.8
